packages/atlas/src/providers/transactions/transactions.manager.tsx

```diff
@@ -97,13 +100,23 @@ export const TransactionsManager: FC = () => {
     setShowFistMintDialog(false)
   }
 
+  const handleTransactionModalClose = () => {
+    if (firstNonMinimizedTransaction) {
+      removeTransaction(firstNonMinimizedTransaction.id)
+    }
+    if (shouldContinueYppFlow) {
+      setShouldContinueYppFlow(false)
+      setShowConnectToYoutubeDialog(true)
+    }
```

Comment:
Do we really need to make TransactionManager aware of YPP stuff? That seems like unnecessary coupling tbh. Couldn't we stick that logic as a callback after creating channel? I think that's the only time we want to display this youtube dialog?

In general, I'm not sure why we need all this additional state, couldn't we just show the youtube dialog always when the user creates a channel?

Reply:
hmm, I guess this `Connect to youtube` dialog could be outside the Ypp authorization modal and the flow will look like this: 
1. User creates the channel. 
2. After the transaction is done. We wait two seconds for the closing transaction modal and then we're opening the `Connect to youtube` dialog 
3. User agrees to connect to youtube, and gets redirected to `studio/ypp`. Immediately after that first step of the authorization modal is showing up
4. User continues the flow. 

Let me know what you think. I would need to do a small refactor, and I need to know that we're on the same page, before I start doing that.

Reply:
yes, that was my thinking, I don't think `Connect to youtube` needs to be part of ypp sign up flow